frontend/src/components/inputs/linkedinloginbutton/LinkedInLoginButton.tsx

```diff
@@ -0,0 +1,25 @@
+import { Button } from "@mantine/core";
+import { useLinkedIn } from "react-linkedin-login-oauth2";
+
+const client_id = "78c9i5swfp7ii3";
```

Comment:
Is it safe to have this client_id in the code? Can anyone do anything with this? It doesn't look very secret like, so just checking

Reply:
Yeah it should be ok, the secret is the only thing that is crucially secret.